### ISSUE TITLE
[BLUS30762] Battlefield 3 security office completely missing

### ISSUE THREAD

**Issue by Darkhost1999** [RPCS3/rpcs3#11862]
I would describe the issue but that's the issue is that there is nothing there to describe. Except a street lamp and dirt.
Essentially there is a crater where a building should be.
[PS3 Video](https://media.discordapp.net/attachments/277227681836302338/967644345694048266/Desktop_2022.04.23_-_23.23.39.09_Trim.mp4)
[RPCS3 Video](https://media.discordapp.net/attachments/277227681836302338/967644342376345630/Desktop_2022.04.23_-_22.45.19.06_Trim.mp4)
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/8549069/RPCS3.log)
[RSX Capture](https://mega.nz/file/OoBhRCyB#30nvb1zDf2g4UOXPmqLofmOclcAGsfyov330qWaa-OE)
[BLUS30762_SAVE](https://github.com/RPCS3/rpcs3/files/8549085/BLUS30762-PROF_SAVE.zip)
If the savedata carries the checkpoint with it all you have to do is follow a singular onscreen button prompt and you're done. You can fully test in approximately 2 minutes.
WCB is enabled to fix missing floor and other textures. I don't want to mess with a lot of settings cause the game isn't stable. I attempted to ask for help on discord was directed to media. Besides I don't think anyone has ever dared to play this far through this game on RPCS3.

**Comment by Darkhost1999** [comment]
For what it's worth OpenGL also has no building
![image](https://user-images.githubusercontent.com/60384196/170163259-5f6a3166-2db6-43f8-a812-3992d53775ae.png)
That ball of white is a car the pink is my team mates, the green is the ground. The right half is what GTAV looks like without disable zcull occlusion queries but rainbow colors was reserved for the left.
TLDR: If I'm supposed to be inside a security office why do I see the sky?
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/8767317/RPCS3.log.gz)

p.s. I'm not missing files
![image](https://user-images.githubusercontent.com/60384196/170164380-f88c5cff-955e-4607-9459-5e2b240befa3.png)

**Comment by Darkhost1999** [comment]
The emulator is a WIP right? So this isn't an issue. I mean who on earth is going to test this far in a game even if I provided a save file so all they have to do is push R1.

**Comment by Megamouse** [comment]
Why shouldn't this be an issue?
I'm sure kd-11 will take a look eventually

**Comment by JimScript** [comment]
It certainly looks like a legit issue, but it is important to be patient as there are a lot issues with this emulator. These issues may not be solved for a long while, but they are important in making this emulator better, so it is good to keep them open. You know looking at the videos, it looks like the building mesh isn't being loaded at all, because you are walking on the ground rather than an invisible building. Maybe there is a script that isn't translated well?

**Comment by Darkhost1999** [comment]
> It certainly looks like a legit issue, but it is important to be patient as there are a lot issues with this emulator. These issues may not be solved for a long while, but they are important in making this emulator better, so it is good to keep them open. You know looking at the videos, it looks like the building mesh isn't being loaded at all, because you are walking on the ground rather than an invisible building. Maybe there is a script that isn't translated well?

I mean how does an entire building go missing without the only logical thing being my game files are messed up. But they test valid.

**Comment by JimScript** [comment]
Well besides the data files being messed up, it could be that the game/emulator is not loading them in properly. In the RPCS3 video the intro sequence doesn't even try to play. Does this building do anything special like have destructible walls or something like what I heard the frostbite engine does, or is it really just normal level geometry? For an emulator like RPCS3, errors like this can occur even if the dumped game files are picture perfect.

**Comment by JimScript** [comment]
Yeah, the building didn't look like it was anything special. It would be great if we could just run the game directly off a compatible Blu-ray drive, but the game files are encrypted and need to be decrypted to be run. But when they are decrypted, all of the files should be there(and in your case, are), so any gameplay affecting bugs should be on the emulators part. And the only time I ever saw the RSX affect gameplay is in Modnation Racers/Littlebigplanet Karting when the dynamic heightmap gets messed up. So I think it might be a Cell issue, have you tried any interpreters to see if the issue stops?

**Comment by JimScript** [comment]
Maybe the level intro scripting is culling things it shouldn't and the interpreters are slowing down the game long enough to see? It is hard to see with the green fog, but they seem to disappear as soon as the second subtitle does. Maybe those 3 things are suppose to spawn in rather than spawn out, and RPCS3 has reversed the script somehow

**Comment by Darkhost1999** [comment]
Those 3 items are no longer required. They could do anything. I'm just not supposed to be able to see any of them to know what they do.

**Comment by Darkhost1999** [comment]
Fixed. IDK When
![BATTLEFIELD 3](https://user-images.githubusercontent.com/60384196/188960667-5eb002b5-3021-4388-8259-32d8e27957eb.png)
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/9509051/RPCS3.log.gz)
